### PR TITLE
[ADD] Blanket order documentation

### PR DESCRIPTION
**Pull request by matt454357**
* Add chapter documenting the use of Blanket Order type Purchase Agreements
* Clarify the proper usage and behavior of dates for Purchase Tender type Purchase Agreements

### CODE REVIEW

**Comment by sle-odoo** [comment]
@lap-odoo could you have a look?

File: purchase/purchases/tender/manage_blanket_orders.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+.. image:: media/manage_multiple_offers00.png
+	:align: center
+
+Set Product Supplier and Lead Time
+----------------------------------
+
+Set up the supplier on the products that will be added to this blanket order.  Be sure to specify the prearranged **Delivery Lead Time**.
+
+For instructions, please read the chapter `How to set several suppliers on a product? <https://www.odoo.com/documentation/user/10.0/purchase/purchases/master/suppliers.html>`__
```
Comment (mart-e) [inline purchase/purchases/tender/manage_blanket_orders.rst]: use relative link, `../purchases/master/suppliers`

File: purchase/purchases/tender/manage_blanket_orders.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
+=========================================
+How to manage Blanket Purchase Agreements
+=========================================
+
+A **Blanket Purchase Order** (BPO), also known as Blanket Purchase Agreement or Call-Off Order, is a contract between you (the customer) and your supplier.  It is used to negotiate a discounted price.  The supplier is benefited by the economies of scale inherent in a large order.  You are benefited by being allowed to take multiple smaller deliveries over a period of time, at a lower price, without paying for the large order immediately.  Each small periodic delivery is called a release or call-off.
```
Comment (mart-e) [inline purchase/purchases/tender/manage_blanket_orders.rst]: keep 80 chars (+/-) for the doc, it makes it easier to review

File: purchase/purchases/tender/manage_blanket_orders.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+For instructions, please read the chapter `How to set several suppliers on a product? <https://www.odoo.com/documentation/user/10.0/purchase/purchases/master/suppliers.html>`__
+
+Create a Blanket Purchase Order
+-------------------------------
+
+To create a new Blanket Purchase Order, open :menuselection:`Purchase --> Purchase Agreements
+(PA)`.
+
+.. image:: media/manage_multiple_offers01.png
```
Comment (mart-e) [inline purchase/purchases/tender/manage_blanket_orders.rst]: Did you forget to add the media?
Keep the format `manage_blanket_orders<number>.png`
Also don't forget to compress it using tools like [pngquant](https://pngquant.org/)

File: purchase/purchases/tender/manage_blanket_orders.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+You will probably need to change the **Order Date** on the RFQ to today.  Odoo Version 10 sets the RFQ **Order Date** equal to the BPO **Agreement Deadline** date.  Future versions of Odoo may set the default **Order Date** as today, for blanket orders.
+
+Be careful to change the **Quantity** field on the RFQ.  By default, the RFQ quantity will be for the entire remaining quantity.  Your individual releases should be for some smaller quantity.
+
+.. image:: media/manage_multiple_offers03.png
+	:align: center
+
+When all of the releases (purchase orders) have been delivered and paid, you can click on **Validate** and **Done**.
+
+View `Purchase Agreements <https://demo.odoo.com/?module=purchase_requisition.action_purchase_requisition>`__
```
Comment (mart-e) [inline purchase/purchases/tender/manage_blanket_orders.rst]: you can use the tag
```rst
.. demo:action:: purchase_requisition.action_purchase_requisition
```

**Comment by tivisse** [comment]
Hi @lap-odoo 

Did you find some time to review the functional content of this documentation page about blanket orders ? 

I would like to merge it after your green light.

Thanks.

**Comment by tivisse** [comment]
Hello @matt454357 

After a functional review from @lap-odoo your documentation page has been merged in 11.0 at https://github.com/odoo/documentation-user/commit/13d8e540f75181b56b2f6e27a7fd77d4ad8c6497

We usually don't add new pages on older versions. And the content is the same for blanket orders (except the setting name).

Have a nice day and thanks for the feedback.

Yannick.

**Pull request closed without merging**